### DISCUSSION TITLE
Double click on free space to navigate up the directory

### DISCUSSION THREAD

**Discussion by carun** [doublecmd/doublecmd#1958] [General]
Looks like this feature was requested a long time here, but it appears to be unanswered at https://doublecmd.h1n.ru/viewtopic.php?t=1767

Is this feature already available and if so, how do I activate this? I'm on latest snapshot build.

**Comment by antodc** [comment]
You can activate it in Settings -> Files views -> Files views extras

**Reply by carun** [reply]
![image](https://github.com/user-attachments/assets/9034099b-1553-4083-8790-64c4337d6ee9)
